src/puppetlabs/trapperkeeper/config.clj

```diff
@@ -59,6 +59,19 @@
     #{".yaml" ".yml"}
     (yaml/parse-string (slurp file))))
 
+(defn override-restart-file-from-cli-data
+  [config-data cli-data]
+  (if-let [cli-restart-file (:restart-file cli-data)]
+    (do
+      (if (get-in config-data [:global :restart-file])
+        (log/warnf
+         (str
+          "restart-file setting specified both on command-line and "
+          "in config file, using command-line value: '%s'")
+         cli-restart-file))
```

Comment:
I made a comment as part of a review and then deleted the comment but there was a notification that it was made, so ... just in case: I thought that there needed to be a `format` instead of `str` but I missed the call to `log/warnf`, so this should be fine. Also it doesn't look like we've got any i18n stuff going on yet in this repo.

Also github--